tests/VcardTest.php

```diff
@@ -133,4 +134,54 @@ public function vcard_with_fn_photo(): void
                 ->photo('data:image/jpeg;base64,'.base64_encode(file_get_contents(__DIR__.'/stubs/photo.jpg')))
         );
     }
+
+    /** @test */
+    public function vcard_with_fn_org(): void
+    {
+        $this->assertMatchesVcardSnapshot(
+            Vcard::make()
+                 ->fullName('John Adam Smith')
+                 ->org('Google;GMail Team;Spam Detection Squad')
+        );
+    }
+
+    /** @test */
+    public function vcard_with_fn_role(): void
+    {
+        $this->assertMatchesVcardSnapshot(
+            Vcard::make()
+                 ->fullName('John Adam Smith')
+                 ->role('Excecutive')
+        );
+    }
+
+    /** @test */
+    public function vcard_with_fn_title(): void
+    {
+        $this->assertMatchesVcardSnapshot(
+            Vcard::make()
+                 ->fullName('John Adam Smith')
+                 ->title('V. P. Research and Development')
+        );
+    }
+
+    /** @test */
+    public function vcard_with_fn_member_mail(): void
+    {
+        $this->assertMatchesVcardSnapshot(
+            Vcard::make()
+                 ->fullName('John Adam Smith')
+                 ->member('mailto:john.smith@company.com')
```

Comment:
WIth the new two optional argument signature this would get `member('john.smith@company.com')` and the UUID one will get `member(null, '550e8400-e29b-11d4-a716-446655440000')` or `member(uuid: '550e8400-e29b-11d4-a716-446655440000')`